tests/test_clients.py

```diff
@@ -35,3 +36,53 @@ def test_close_without_clients(hlwm):
 def test_close(hlwm, running_clients_num):
     hlwm.create_clients(running_clients_num)
     hlwm.call('close')
+
+
+@pytest.mark.filterwarnings("ignore:tostring")
+@pytest.mark.parametrize("urgent", [True, False])
+def test_urgent_on_start(hlwm, x11, urgent):
+    command = []
+    if urgent:
+        # prepent urgent command
+        command.append(r"echo -e '\a'")
+    command += [
```

Comment:
Why not use `command.append()` here, too?

Reply:
Ah you're still working on this anyway. My bad, nevermind.

Reply:
I expected this list to become longer... Also, I hoped that the race condition is not triggered in travis. I was wrong, and so I've replaced this part.